### PR TITLE
Adds ways to get involved.

### PR DESCRIPTION
**Pull request by coollog**
Preview at: https://github.com/GoogleContainerTools/jib/tree/coollog-patch-2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,7 +45,15 @@ See also [rules_docker](https://github.com/bazelbuild/rules_docker) for a simila
 
 See the [Frequently Asked Questions (FAQ) wiki page](/../../wiki/Frequently-Asked-Questions-(FAQ)).
 
-## Community
+## Get involved with the community
 
+We welcome contributions! Here's how you can contribute:
+
+* Browse [issues](https://github.com/GoogleContainerTools/jib/issues) or file [an issue](https://github.com/GoogleContainerTools/jib/issues/new)
```
Comment (briandealwis) [inline README.md]: Seems a bit better to include the verb in the link? _Browse issues_ or _file an issue_. What do you think?

File: README.md [hunk 2]
```diff
@@ -45,7 +45,15 @@ See also [rules_docker](https://github.com/bazelbuild/rules_docker) for a simila
...
 * Chat with us on [gitter](https://gitter.im/google/jib)
-* [jib-users mailing list](https://groups.google.com/forum/#!forum/jib-users)
+* Join the [jib-users mailing list](https://groups.google.com/forum/#!forum/jib-users)
+* Contribute:
+  * Try to fix [good first issues](https://github.com/GoogleContainerTools/jib/labels/good%20first%20issue)
+  * Help out on [issues that need help](https://github.com/GoogleContainerTools/jib/labels/help%20wanted)
+  * Join in on [discussion issues](https://github.com/GoogleContainerTools/jib/labels/discuss)
+<!--  * Read the [style guide] -->
```
Comment (briandealwis) [inline README.md]: Should include a link to the code of conduct?

File: README.md (hunk 1 above)
Comment (coollog) [reply README.md]: Sounds good!

File: README.md (hunk 2 above)
Comment (coollog) [reply README.md]: Added!

File: README.md [hunk 3]
```diff
@@ -46,7 +46,16 @@ See also [rules_docker](https://github.com/bazelbuild/rules_docker) for a simila
...
 
-## Community
+## Get involved with the community
 
+We welcome contributions! Here's how you can contribute:
+
+* [Browse issues](https://github.com/GoogleContainerTools/jib/issues) or [file an issue](https://github.com/GoogleContainerTools/jib/issues/new)
 * Chat with us on [gitter](https://gitter.im/google/jib)
-* [jib-users mailing list](https://groups.google.com/forum/#!forum/jib-users)
+* Join the [jib-users mailing list](https://groups.google.com/forum/#!forum/jib-users)
```
Comment (TadCordle) [inline README.md]: Do we even use this?

**Pull request merged**